pkgs/servers/prowlarr/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchurl, mono, libmediainfo, sqlite, curl, makeWrapper, icu, dotnetCorePackages, openssl, nixosTests }:
+
+let
+  os = if stdenv.isDarwin then "osx" else "linux";
```

Comment:
We can't assume that else it is linux. It should throw if it is not darwin or linux.

Reply:
Ah agreed, sounds good!